apps/igloo-kit-cli/src/cli/logger.rs

```diff
@@ -1,3 +1,39 @@
+//! # Logger Module
```

Comment:
That's interesting - I would have put this documentation on the documentation website in the section for contributors.

Since we don't have that up - we can leave it here for now and revisit later.